src/test/test-sbat.c

```diff
@@ -13,7 +13,7 @@ TEST(sbat_section_text) {
                  "%s"
                  "------------------>&-----------------------------------------",
```

Comment:
or those supposed to be scissors? unicode has a chracter for that, ffs: 

✀

(only right facing unfortunately, but still!)

Reply:
Don't look at me, it was already like that when I arrived!

Reply:
This is a traditional thing used in kernel patches and docs. I think it's cute. Please note that the scissors ascii-art appears twice, the second time below, mirrored. So please either change both or neither.

Reply:
> So please either change both or neither.

there's no char for the other direction, so I'll drop it then

Reply:
now i am sad.